database/src/main/scala/cromwell/database/slick/SlickDatabase.scala

```diff
@@ -602,7 +602,7 @@ class SlickDatabase(databaseConfig: Config) extends SqlDatabase {
     runTransaction(action)
   }
 
-  protected def addMetadata(events: Iterable[Metadatum])
+  override def addMetadata(events: Iterable[Metadatum])
```

Comment:
I don't understand the new database structure at all so feel free to push back... But I thought these would be moved to a DataAccess in the services package, and `database` would only contain very generic things?